### ISSUE TITLE
Hover trigger

### ISSUE THREAD

**Issue by Jojoker33** [elbwalker/walker.js#67]
**Is your feature request related to a problem? Please describe.**
The addition of  an hover trigger that fires when interaction with components, elements wich expand, show other element such as submenu’s by example 

**Describe the solution you'd like**
With almost the same usage as the already existing click trigger, but it will track only the _onmouseenter_ event on the wanted target

**Comment by alexanderkirtzel** [comment]
Good idea. Thinking about `elbaction="hover"` that triggers multiple times & immediately on hovering. Would that be your expected behaviour?

It could result in a bunch of events. Later we could add some configuration to control the frequency. But it should be somehow consistent with visible that only fires once.

Alternative idea: Three triggers
- hover: only once (consistent with visible)
- mouseenter: every time like onmouseenter
- mouseleave: every time like onmouseleave

**Comment by Jojoker33** [comment]
> Good idea. Thinking about `elbaction="hover"` that triggers multiple times & immediately on hovering. Would that be your expected behaviour?
> 
> It could result in a bunch of events. Later we could add some configuration to control the frequency. But it should be somehow consistent with visible that only fires once.
> 
> Alternative idea: Three triggers
> 
> * hover: only once (consistent with visible)
> * mouseenter: every time like onmouseenter
> * mouseleave: every time like onmouseleave

I was expected more the second one of the three triggers ( =>   mouseenter: every time like onmouseenter )  as I have already an use case in mind for it but your solution with frequency configuration looks like a very awesome feature to have :-)
